src/components/CurrentUser/CurrentUser.jsx

```diff
@@ -1,12 +1,64 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './CurrentUser.scss';
 
-export const CurrentUser = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+export class CurrentUser extends React.Component {
+  state = {
+    user: {},
+  }
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+  async componentDidMount() {
+    const user = await this.getUser();
+
+    this.setUser(user);
+  }
+
+  async componentDidUpdate(prevProps) {
+    if (prevProps.userId !== this.props.userId) {
+      const user = await this.getUser();
+
+      this.setUser(user);
+    }
+  }
+
+  clear = () => {
+    this.setState({ user: {} });
+  }
```

Comment:
after pressing clear button, user can't choose todo with the same user id. If you want reset info about user, you need to set default(0) value to selectedUserId